### PR TITLE
python3Packages.shellingham: 1.3.2 -> 1.4.0

### PR DESCRIPTION
**Pull request by fabaff**
###### Motivation for this change
Update to latest upstream release 1.4.0

Change log: https://github.com/sarugaku/shellingham/blob/master/CHANGELOG.rst

- Enable tests

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Relase notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).

Result of `nixpkgs-review pr 129628` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>

  <summary>21 packages failed to build and already failed to build on hydra master:</summary>
  <ul>
<li>esphome: log was empty</li>
<li>lexicon: log was empty</li>
<li>markdown-anki-decks: log was empty</li>
<li>platformio: log was empty</li>
<li>python38Packages.authcaptureproxy: log was empty</li>
<li>python38Packages.gradient: log was empty</li>
<li>python38Packages.pathy: log was empty</li>
<li>python38Packages.spacy: log was empty</li>
<li>python38Packages.spacy-transformers: log was empty</li>
<li>staticjinja: log was empty</li>
<li>python38Packages.teslajsonpy: log was empty</li>
<li>python38Packages.textacy: log was empty</li>
<li>python38Packages.toggl-cli: log was empty</li>
<li>python38Packages.tvdb_api: log was empty</li>
<li>python38Packages.typer: log was empty</li>
<li>python39Packages.authcaptureproxy: log was empty</li>
<li>python39Packages.gradient: log was empty</li>
<li>python39Packages.staticjinja: log was empty</li>
<li>python39Packages.teslajsonpy: log was empty</li>
<li>python39Packages.tvdb_api: log was empty</li>
<li>python39Packages.typer: log was empty</li>
  </ul>
</details>
<details>

  <summary>1 package failed to build and are new build failure:</summary>
  <ul>
<li>(python38Packages.staticjinja): log was empty</li>
  </ul>
</details>
<details>
<summary>
46 packages built:
</summary>
<ul>
<li>cmake-language-server</li>
<li>cryptop</li>
<li>doitlive</li>
<li>home-assistant</li>
<li>jrnl</li>
<li>python38Packages.aioeafm</li>
<li>python38Packages.aiomodernforms</li>
<li>python38Packages.aria2p</li>
<li>python38Packages.backoff</li>
<li>python38Packages.censys</li>
<li>python38Packages.click-completion</li>
<li>python38Packages.hypothesis-auto</li>
<li>python38Packages.myjwt</li>
<li>python38Packages.poetry</li>
<li>python38Packages.pyairnow</li>
<li>python38Packages.pysnow</li>
<li>python38Packages.pysychonaut</li>
<li>python38Packages.python-miio</li>
<li>python38Packages.questionary</li>
<li>python38Packages.requests-cache</li>
<li>python38Packages.shellingham</li>
<li>python38Packages.toonapi</li>
<li>python38Packages.url-normalize</li>
<li>python38Packages.wled</li>
<li>python38Packages.yalesmartalarmclient</li>
<li>python39Packages.aioeafm</li>
<li>python39Packages.aiomodernforms</li>
<li>python39Packages.backoff</li>
<li>python39Packages.censys</li>
<li>python39Packages.click-completion</li>
<li>python39Packages.hypothesis-auto</li>
<li>python39Packages.myjwt</li>
<li>python39Packages.poetry</li>
<li>python39Packages.pyairnow</li>
<li>python39Packages.pysnow</li>
<li>python39Packages.pysychonaut</li>
<li>python39Packages.python-miio</li>
<li>python39Packages.questionary</li>
<li>python39Packages.requests-cache</li>
<li>python39Packages.shellingham</li>
<li>python39Packages.toonapi</li>
<li>python39Packages.url-normalize</li>
<li>python39Packages.wled</li>
<li>python39Packages.yalesmartalarmclient</li>
<li>theharvester</li>
<li>tvnamer</li>
</ul>
</details>
